ansible_collections/arista/avd/tests/integration/targets/configlet_build_config/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+testcase: '*'
+inventory_path: "/../../../../../../../../inventory/"
```

Comment:
you are starting with `/` making this an absolute path. But below it is inserted in the middle of a path, resulting in double `/` before and after. Do we really need this l extremely long relative path?

Reply:
actually in run time ansible-test is creating a temp path and refering to that only , thats why we need to take this path.

Reply:
There must be some variable set by Ansible, pointing to this place. The very long relative path is likely to fail between different environments.
Check out [this article](https://docs.ansible.com/ansible/latest/reference_appendices/special_variables.html), if you can find a variable that covers this.

Reply:
Okay, thanks!

Reply:
Will check it once.

Reply:
relative paths have been updated.